lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -342,7 +342,12 @@ def get_client( self, job_destination_params, job_id, env=[] ):
 
         encoded_job_id = self.app.security.encode_id(job_id)
         job_key = self.app.security.encode_id( job_id, kind="jobs_files" )
-        files_endpoint = "%s/api/jobs/%s/files?job_key=%s" % (
+        endpoint_base = "%s/api/jobs/%s?job_key=%s"
```

Comment:
We broke non-nginx uploads here. The ``files`` got lost in the endpoint description.